command-parser.js

```diff
@@ -7,7 +7,7 @@
  *
  * Individual commands are put in:
  *   commands.js - "core" commands that shouldn't be modified
- *   config/commands.js - other commands that can be safely modified
+ *   chat-plugins/info.js - other commands that can be safely modified
  *
  * The command API is (mostly) documented in config/commands.js
```

Comment:
The examples are written to use all commands from ```config/commands.js```. Should they be changed to use example commands from ```commands.js``` instead?